aspnetcore/client-side/bundling-and-minification.md

```diff
@@ -265,8 +265,16 @@ If Visual Studio and/or the Bundler & Minifier extension aren't available, conve
 
 Add a *package.json* file, with the following `devDependencies`, to the project root:
 
+# [Gulp 4.x](#tab/gulp4x/)
+
+[!code-json[](../client-side/bundling-and-minification/samples/BuildBundlerMinifierApp/package2.json?range=5-13)]
+
+# [Gulp 3.x](#tab/gulp3x/)
```

Comment:
Please remove the Gulp 3.x content all throughout. Showing only 4.x is fine.

Reply:
So remove the notion of tab usage here for versioning of Gulp all together? Basically, just update the existing stuff with Gulp v4 removing v3 entirely - right?

Reply:
That's correct. Thank you for the help!

Reply:
Sounds good, will update soon. Thank you 🤘